docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/configuring-autoscaling.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Configuring Grafana Mimir autoscaling with Jsonnet"
+menuTitle: "Configuring autoscaling"
+description: "Learn how to configure Grafana Mimir autoscaling when using Jsonnet."
+weight: 30
+---
+
+# Configuring Grafana Mimir autoscaling with Jsonnet
+
+Mimir Jsonnet supports autoscaling for the following components:
+
+- [Querier]({{< relref "../../architecture/components/querier.md" >}})
+
+Autoscaling is implemented using Kubernetes Horizontal Pod Autoscaler (HPA) and is based on Prometheus metrics and [KEDA](https://keda.sh).
```

Comment:
```suggestion
Autoscaling, which is based on Prometheus metrics and [KEDA (Kubernetes-based Event Driven Autoscaler)](https://keda.sh), uses Kubernetes’ Horizontal Pod Autoscaler (HPA).
```
* Do we care that is implemented using or simply uses?

Reply:
Do you expect the user to be more familiar with "KEDA" or "Kubernetes-based Event Driven Autoscaler"?